# Samuel Miravet-Verde personal page

### Research 

<img style="float: left; width: 15%; height: auto" src="./profile_image.jpeg">

Samuel Miravet-​Verde is a computational biologist with expertise in microbiology and artificial intelligence-​based bioinformatic approaches to understand complex microbiological genomic and functional aspects. Throughout his PhD he led research to show that small proteins are as important as larger proteins in microbial systems, developing multiple tools to study them, as well as other problems such as assessing gene essentiality, and as main data-​analyst in several collaborative projects. He joined the Sunagawa lab as a postdoc in 2022 to work in the development of the Ocean Microbiome Database and projects deriving from this resource.

### Experience 

- 2022 - current: **Postdoc** at the Sunagawa lab. in ETH Zurich (Zurich, Switzerland)
- 2021-​2022: **Junior Postdoc**, Centre for Genomic Regulation (Barcelona, Spain)
- 2017-​2017: **Visiting PhD**, Karr lab. at Icahn Institute for Data Science and Genomic Technology (New York, United States)
- 2016-​2021: **PhD** in Biomedicine and Computational Biology, Serrano lab. at Centre for Genomic Regulation (Barcelona, Spain)
- 2015-​2016: **Bioinformatics Intern**,  Serrano lab. at Centre for Genomic Regulation (Barcelona, Spain)
- 2013-​2014: **Bioinformatics Intern**, GEM Biosoft (Valencia, Spain)
- 2013-​2014: **Bioinformatics Intern**, Foundation for the Promotion of Health and Biomedical Research (Valencia, Spain)
- 2011-​2013: **Research Intern**, Foundation for the Promotion of Health and Biomedical Research (Valencia, Spain)

### Education

- ​Oct. 2016 - May 2021: **PhD** in Biomedicine and Computational Biology, Universitat Pompeu Fabra (Barcelona, Spain)
- Sep. 2014 - Jun. ​2016: **MSc** in Bioinformatics for Health Sciences, Universitat Pompeu Fabra (Barcelona, Spain)
- Sep. 2010 - Jun. ​2014: **BSc** in Biochemistry and Biomedical Sciences, Universitat de València (Valencia, Spain)

### Publications

As first author:

- Miravet-Verde S; Burgos R; Delgado J; Lluch-Senar M; Serrano L, 2020. FASTQINS and ANUBIS: two bioinformatic tools to explore facts and artifacts in transposon sequencing and essentiality studies.  Nucleic Acids Res 48(17):e102
- Miravet-Verde S; Ferrar T; Espadas-García G; Mazzolini R; Gharrab A; Sabido E; Serrano L; Lluch-Senar M, 2019. Unraveling the hidden universe of small proteins in bacterial genomes.  Mol Syst Biol 15(2):e8290
- Miravet-Verde S; Lloréns-Rico V; Serrano L, 2017. Alternative transcriptional regulation in genome-reduced bacteria.  Curr Opin Microbiol 39:89-95

As main data analyst:

- Piñero-Lambea C; Garcia-Ramallo E; Miravet-Verde S; Burgos R; Scarpa M; Serrano L; Lluch-Senar M, 2022. SURE editing: combining oligo-recombineering and programmable insertion/deletion of selection markers to efficiently edit the Mycoplasma pneumoniae genome. Nucleic Acids Res 50 (22)
- Shaw D; Miravet-Verde S; Piñero-Lambea C; Serrano L; Lluch-Senar M, 2021. LoxTnSeq: random transposon insertions combined with cre/lox recombination and counterselection to generate large random genome reductions.  Microb Biotechnol 14(6):2403-2419
- Montero-Blay A; Piñero-Lambea C; Miravet-Verde S; Lluch-Senar M; Serrano L, 2020. Inferring Active Metabolic Pathways from Proteomics and Essentiality Data.  Cell Rep 31(9):107722
- Montero-Blay A; Miravet-Verde S; Lluch-Senar M; Piñero-Lambea C; Serrano L, 2019. SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes.  DNA Res 26(4):327-339

As supervisor:

- Anglada-Girotto M; Miravet-Verde S; Serrano L; Head SA, 2022. robustica: customizable robust independent component analysis. BMC bioinformatics 23 (1), 1-9

Other collaborations:

- Ciampi L; Mantica F; López-Blanch L; Permanyer J; Rodriguez-Marín C; Zang J; Cianferoni D; Jiménez-Delgado S; Bonnal S; Miravet-Verde S; Ruprecht V; CF Neuhauss S; Banfi S; Carrella S; Serrano L; Head SA; Irimia M, 2022. Specialization of the photoreceptor transcriptome by Srrm3-dependent microexons is required for outer segment maintenance and vision. PNAS 119 (29), e2117090119
- Broto A; Gaspari E; Miravet-Verde S; Dos Santos VAPM; Isalan M, 2022. A genetic toolkit and gene switches to limit Mycoplasma growth for biosafety applications.  Nat Commun 13(1):1910
- Martín-Pardillos A; Valls Chiva Á; Bande Vargas G; Hurtado Blanco P; Piñeiro Cid R; Guijarro PJ; Hümmer S; Bejar Serrano E; Rodriguez-Casanova A; Diaz-Lagares Á; Castellvi J; Miravet-Verde S; Serrano L; Lluch-Senar M; Sebastian V; Bribian A; López-Mascaraque L; López-López R; Ramón Y Cajal S, 2019. The role of clonal communication and heterogeneity in breast cancer.  BMC Cancer 19(1):666
- Gosalbes MJ; Vallès Y; Jiménez-Hernández N; Balle C; Riva P; Miravet-Verde S; de Vries LE; Llop S; Agersø Y; Sørensen SJ; Ballester F; Francino MP, 2016. High frequencies of antibiotic resistance genes in infants' meconium and early fecal samples.  J Dev Orig Health Dis 7(1):35-44
- Dorado-Morales P; Iglesias A; Zafrilla G; Valero A; Torres A; Miravet-Verde S; de Loma J; Mañas M; Ruiz A; Corman A; Morales LJ; Peretó J; Vilanova C; Porcar M, 2014. Engineering bacteria to form a biofilm and induce clumping in Caenorhabditis elegans.  ACS Synth Biol 3(12):941-3